app/src/androidTest/java/ch/epfl/polychef/EitherTest.java

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.polychef;
```

Comment:
This class was refactored by @Ingloryon in the utils package, this is a duplicate you should remove it